### PR TITLE
UI Refactor - Use more of MOJ Frontend

### PR DESCRIPTION
**Pull request by aliuk2012**
# Context

# Changes in this PR

- Replace custom datepicker with official MOJ DataPicker
- Replace hardcoded button menu markup with offical MOJ button menu component

No

### Before
image

### After
image

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

File: server/views/temporary-accommodation/reports/index.njk [hunk 1]
```diff
@@ -2,14 +2,13 @@
 {% extends "../../partials/layout.njk" %}
 {% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
 {% from "../../components/formFields/form-page-select/macro.njk" import formPageSelect %}
-{% from "../../components/datePicker/macro.njk" import hmppsDatepicker %}
+{%- from "moj/components/date-picker/macro.njk" import mojDatePicker -%}
```
Comment (Rosie-Brigham) [inline server/views/temporary-accommodation/reports/index.njk:5]: 💃

File: server/views/temporary-accommodation/reports/index.njk [hunk 2]
```diff
@@ -2,14 +2,13 @@
...
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 
 {% set pageTitle = "Reports - " + applicationName %}
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block extraScripts %}
-    <script type="text/javascript" nonce="{{ cspNonce }}" src="/assets/js/datepicker.js"></script>
```
Comment (Rosie-Brigham) [inline server/views/temporary-accommodation/reports/index.njk:12]: Assume this comes packaged with the MOJ date picker?

**Review by Rosie-Brigham — APPROVED** [review]
🔥 !

File: server/views/temporary-accommodation/reports/index.njk (hunk 2 above)
Comment (aliuk2012) [reply server/views/temporary-accommodation/reports/index.njk:12]: Thats correct

**Pull request merged**